guide/daily-spending-wallet/sending.md

```diff
@@ -292,6 +305,20 @@ It is ideal when the application can automatically identify and fix or avoid the
 
 Effectively supporting users when problems occur can build trust and confidence, and essential aspect for financial applications.
 
+### Encouraging lightning network
+
+For the majority of payments a user makes, Lightning is likely to be the best option. It will be faster, more private, and cost less. An ideal scenario would be one where the user does not spend time considering whether to pay on-chain or Lightning — it's all bitcoin to them.
```

Comment:
```suggestion
Lightning is likely to be the best option for the majority of payments a user makes. It will be faster, more private, and cost less. An ideal scenario would be where the user does not spend time considering whether to pay on-chain or Lightning — it's all bitcoin to them.
```